ansible/configs/kni-osp/files/cloud_providers/osp_cloud_template_master.j2

```diff
@@ -0,0 +1,262 @@
+heat_template_version: 2018-03-02
```

Comment:
If you are using the OSP cloud provider and need to use your own HOT, please see the changes merged in https://github.com/redhat-cop/agnosticd/pull/1763/files.

We had to move some of the resource creation out of the HOT to make sure resources got created by the correct user (admin vs user).

Reply:
most of the changes there are meant to allow for some nodes having  2 interfaces and have th eability to turn posrt security off on some of those. Since I'm not creating any role/project or anything that should need admin permissions (which I don't have when testing in my OSP Sandbox environment anyway), this shouldn't be an issue, or is it ?